exercises/yacht/Example.fs

```diff
@@ -14,6 +14,21 @@ type Category =
     | Choice
     | Yacht
 
+type DieValue =
```

Comment:
Was there a reason not to name this `Die`? If not, I think I would prefer that (shorter) name.

Reply:
No reason, just what jumped into my head first. Shorter name sounds better -- I'll make the change.